shardingsphere-proxy/shardingsphere-proxy-backend/pom.xml

```diff
@@ -132,5 +132,18 @@
             <artifactId>postgresql</artifactId>
             <scope>runtime</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>1.7.4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito2</artifactId>
+            <version>1.7.4</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Please do not add other dependency if unnecessary, the reason are:
1. It is better to use one mock framework
2. If we need add more mock framework, please add the notice and licenses too